src/kirby/components/icon/custom-icon-name.directive.tns.ts

```diff
@@ -0,0 +1,65 @@
+import { LayoutBase } from 'tns-core-modules/ui/layouts/layout-base';
+import { Label } from 'tns-core-modules/ui/label/label';
+import {
+  Directive,
+  ElementRef,
+  Input,
+  Inject,
+  AfterViewInit,
+  OnChanges,
+  SimpleChanges,
+} from '@angular/core';
+
+import { CUSTOM_FONT_SETTINGS, CustomIconSettings } from './custom-icon-settings';
+
+@Directive({
+  // tslint:disable-next-line:directive-selector
+  selector: '[customName]',
+})
+export class CustomIconNameDirective implements AfterViewInit, OnChanges {
+  // tslint:disable-next-line:no-input-rename
+  @Input('customName') customName: string;
+
+  constructor(
+    private element: ElementRef,
+    @Inject(CUSTOM_FONT_SETTINGS) private customIconSettings: CustomIconSettings
+  ) {}
+
+  public ngAfterViewInit(): void {
+    this.handleCustomIcon(this.customName);
+  }
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    if (changes.customName) {
+      this.handleCustomIcon(this.customName);
+    }
+  }
+
+  private handleCustomIcon(name: string) {
+    if (name && this.customIconSettings[0].icons) {
+      const fontFamily = this.customIconSettings[0].fontfamily;
+      const icon = this.getCustomIcon(name);
```

Comment:
Probably need to have a way of finding the right customIconSettings and then sending that as a parameter to getCustomIcon...

Reply:
Support for multiple custom fonts is removed